admin-dev/themes/default/scss/partials/_commons.scss

```diff
@@ -0,0 +1,246 @@
+// helpers class for fixing input width
+.fixed-width-xs {
+  width: 48px !important;
+}
+.fixed-width-sm {
+  width: 80px !important;
+}
+.fixed-width-md {
+  width: 120px !important;
+}
+.fixed-width-lg {
+  width: 160px !important;
+}
+.fixed-width-xl {
+  width: 200px !important;
+}
+.fixed-width-xxl {
+  width: 250px !important;
+}
+//quick fix for inline hidden content on device width - waiting for bootstrap update https://github.com/twbs/bootstrap/issues/8869
```

Comment:
@NeOMakinG Twitter Bootstrap has been updated ?

Reply:
We can't update it due to SemVer